packages/api-plugin/src/generators/trpc-api-generator/files/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./src/*"]
+    },
+    "target": "es2017",
```

Comment:
i'm wondering if we should update this to be something more recent we have options for `2020` `2021` `2022` and `esnext`?

Reply:
good call, might as well use the latest version we can